src/frequenz/client/base/streaming.py

```diff
@@ -24,15 +23,15 @@
 """The output type of the stream."""
 
 
-class GrpcStreamingHelper(Generic[InputT, OutputT]):
+class Broadcaster(Generic[InputT, OutputT]):
     """Helper class to handle grpc streaming methods."""
```

Comment:
Maybe the word Helper can go away from the docstring as well?

Reply:
I was planning to update docs in a separate PR, I didn't want to have to rewrite all the docs if there was a lot of push back about the new names.